docs/tutorial/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# Use an official Python runtime as a base image
+FROM python:2
+
+# Set the working directory to /app
+WORKDIR /package
```

Comment:
Oh, just realized the comment is probably for educational purposes. Let's keep it around, but maybe update it so it doesn't reference the directory by name? Something like "Set the working directory to the package directory"?